modules/hello-world/pages/platform-help.adoc

```diff
@@ -0,0 +1,172 @@
+= Setting Up Couchbase PHP SDK with VSCode
+:description: pass:q[Discover how to get up and running developing applications with the Couchbase PHP SDK 3.0+ using `Visual Studio Code`.]
+:page-aliases: ROOT:platform-introduction,ROOT:platform-help,php-environment
+:navtitle: Setting Up the PHP SDK
+
+[abstract]
+{description}
+
+
+A simple PHP orientation intro for _non-PHP_ folk who are evaluating the Couchbase PHP SDK.
+
+
+[IMPORTANT]
+.Is This Page for You?
+====
+This page is to help evaluate the Couchbase PHP SDK, if PHP is not where you spend the majority of your working day. 
+It is aimed at Software Architects, QE folk, managers, and anyone else who needs to run through using the PHP SDK without necessarily being comfortable with the PHP environment.
+If this is not you, head back to the xref:overview.adoc[rest of the Couchbase PHP SDK documentation].
+====
+
+
+== Installing PHP and the Couchbase PHP SDK
+
+The Couchbase PHP SDK generally tracks PHP's own https://www.php.net/supported-versions.php[supported versions].
+As of April 2022, it is compatible with PHP 7.4, 8.0, and 8.1.
+
+In this section, we'll recommend a known good installation path here for *the purposes of evaluating* Couchbase.
+Though we'll call out some key sections where you may wish to override that for your needs, you and your in-house PHP experts will want to make a decision for ongoing development and deployment.
+
+The https://www.php.net/manual/en/install.php[PHP install page] has full details and many options for installing PHP on various operating systems.
+
+Couchbase generally tracks https://www.php.net/supported-versions.php[PHP supported versions],
+and will recommend using the most up-to-date fully supported version (currently 8.1)
+unless otherwise specified in our xref:project-docs:sdk-release-notes.adoc[release notes] or xref:project-docs:compatibility.adoc[compatibility page].
+
+
+[{tabs}]
+====
+Mac::
++
+--
+Install PHP using Homebrew.
+Once you have installed PHP, simply use `pecl` to install the Couchbase PHP SDK.
+
+
+[source,console]
+----
+brew install php
+pecl install https://packages.couchbase.com/clients/php/couchbase-4.0.0.tgz
+----
+--
+Linux::
++
+--
+https://www.php.net/manual/en/install.unix.php has information on installing on various Linux systems.
+Where there are no specific instructions on the official PHP page, or in our tested guidance below, it is worth consulting your distro's trusted documentation, as there may be known-good repositories with PHP binaries to evaluate.
+
+Once you have installed PHP, simply use `pecl` to install the Couchbase PHP SDK.
+
+[source,console]
+----
+pecl install https://packages.couchbase.com/clients/php/couchbase-4.0.0.tgz
```

Comment:
I thought we were sticking with $ prompt for BASH?

Reply:
Oh, er, possibly. The copy/paste of `$` doesn't feel great, let's discuss, happy to push either way to be consistent 👍